pkg/utils/utils.go

```diff
@@ -87,3 +87,23 @@ func FilterAlerts(data template.Data, selector *v1.LabelSelector, logger log.Log
 
 	return newData
 }
+
+// SplitAlerts splits input data into a slice of data,
+// and each of them only contains one Alert (data.Alerts only contains 1 element).
+// This function serves to break a long message into several short messages.
+func SplitAlerts(data template.Data) []template.Data {
```

Comment:
If this function is not used, please delete it.

Reply:
Removed